--- disemvowel.module.original	2010-08-04 10:51:37.000000000 -0400
+++ disemvowel.module	2010-08-04 10:41:56.000000000 -0400
@@ -163,7 +163,7 @@ function disemvowel_comment_disemvowel($
     $cid = $context['cid'];
     $subject = db_result(db_query("SELECT subject FROM {comments} WHERE cid = %d", $cid));
   }
-  db_query('UPDATE {comments} SET disemvowel = %d WHERE cid = %d', 0, $cid);
+  db_query('UPDATE {comments} SET disemvowel = %d WHERE cid = %d', 1, $cid);
   watchdog('disemvowel', 'Disemvoweled comment %subject.', array('%subject' => $subject));
 }
 
@@ -176,6 +176,6 @@ function disemvowel_comment_emvowel($com
     $cid = $context['cid'];
     $subject = db_result(db_query("SELECT subject FROM {comments} WHERE cid = %d", $cid));
   }
-  db_query('UPDATE {comments} SET disemvowel = %d WHERE cid = %d', 1, $cid);
+  db_query('UPDATE {comments} SET disemvowel = %d WHERE cid = %d', 0, $cid);
   watchdog('disemvowel', 'Emvoweled comment %subject.', array('%subject' => $subject));
 }
